app/src/main/java/io/github/droidkaigi/confsched/activity/MainActivity.java

```diff
@@ -90,6 +92,29 @@ private void initView() {
         toggle.syncState();
         binding.navView.setNavigationItemSelectedListener(this);
         binding.navView.setCheckedItem(R.id.nav_all_sessions);
+
+        binding.getRoot().getViewTreeObserver().addOnGlobalLayoutListener(new ViewTreeObserver.OnGlobalLayoutListener() {
+            @Override
+            public void onGlobalLayout() {
+                if (ViewCompat.isLaidOut(binding.appbar)) {
+                    binding.getRoot().getViewTreeObserver().removeOnGlobalLayoutListener(this);
+                    initAppBarBehavior();
```

Comment:
If this put  `onCreate()` timing, `binding.appbar.getLayoutParams().getBehavior()` is null.
So I wait to ending layout.